internal/app/machined/pkg/runtime/v1alpha1/v1alpha1_sequencer.go

```diff
@@ -413,10 +418,6 @@ func (*Sequencer) StageUpgrade(r runtime.Runtime, in *machineapi.UpgradeRequest)
 		).Append(
 			"dbus",
 			StopDBus,
-		).AppendWhen(
```

Comment:
we don;t need this anymore?

Reply:
yes (and below), as we no longer wipe on ugprade